src/platform/packages/private/kbn-grid-layout/grid/use_grid_layout_events/sensors/mouse.ts

```diff
@@ -43,15 +48,15 @@ export const startMouseInteraction = ({
     onMove(ev);
   };
 
-  const handleEnd = () => {
+  const handleEnd = (ev: Event) => {
     document.removeEventListener('scroll', onMove);
     document.removeEventListener('mousemove', handleMouseMove);
     stopAutoScroll();
-    onEnd();
+    onEnd(ev);
```

Comment:
It doesn't look like `onEnd` uses this argument? (see `src/platform/packages/private/kbn-grid-layout/grid/use_grid_layout_events/panel/events.ts`)